src/js/views/patients/schedule/schedule_views.js

```diff
@@ -206,6 +211,39 @@ const DayListView = CollectionView.extend({
   },
   initialize({ state }) {
     this.state = state;
+
+    this.listenTo(state, 'change:searchQuery', this.searchList);
+  },
+  childViewTriggers: {
+    'render': 'listItem:render',
+  },
+  onListItemRender(view) {
+    const date = dayjs(this.model.get('date'));
+    view.searchString = `${ date.format('D') } ${ date.format('MMM, ddd') } ${ view.$el.text() }`;
+  },
+  searchList(state, searchQuery) {
+    if (!searchQuery) {
+      this.removeFilter();
+      this.triggerMethod('filter');
```

Comment:
As far as I could tell, `removeFilter` does not emit any events, but we need to emit one in order for the parent collection view to know that the searchQuery has been cleared. There might be a better way to do this though.

Reply:
Ah yeah.  Use `render:children` instead of `filter`. The `filter` event only occurs if views are filtered, but if there is no `filter` it doesn't attempt to filter.